File: ios/Classes/service/BleIdentificationServiceCallback.swift
//
//  BleIdentificationServiceCallback.swift
//  flutter_ble_identification
//
//  Created by WOO JIN HWANG on 2022/06/09.
//

import Foundation

protocol BleIdentificationServiceCallback {
  func onAccessSuccess(resultCode: String)
  func onAccessFailure(resultCode: String)
  func onScanError(errorJson: String)
  func onGattError(errorJson: String)
  func onGattInfo(message: String)
}

File: android/src/main/kotlin/com/pravera/flutter_ble_identification/service/BleIdentificationServiceCallback.kt
package com.pravera.flutter_ble_identification.service

interface BleIdentificationServiceCallback {
    fun onAccessSuccess(resultCode: String)
    fun onAccessFailure(resultCode: String)
    fun onScanError(errorJson: String)
    fun onGattError(errorJson: String)
    fun onGattInfo(message: String)
}
